### DISCUSSION TITLE
What happens if you try to dump a fake game?

### DISCUSSION THREAD

**Discussion by bmreading** [sanni/cartreader#888] [General Chat]
I have an open cart reader that I use to make my own backups of NES games. So far, I haven't had any issues with dumping any of my games, but I was wondering if that can be used as a good indicator of whether my games are authentic or not.

Does anyone know if counterfeit games can be dumped using the reader? On a related note, can it harm the hardware if I put a counterfeit game in there by mistake?

**Comment by Ancyker** [comment]
It depends on the system and the game. Many fake GBA games, especially Pokemon games, used modified ROMs so the CRC won't match anything. But most games without save capability won't have any changes made.

With that said, functionality could be added to test if a cart is legit by seeing if it's writable, as legit games aren't.


> Does anyone know if counterfeit games can be dumped using the reader?

Most can be, the only system supported by the base version of HW5 (that is to say, without adapters) that I can see possibly having issues is the NES(/FC) since it lacks a way to know how the cart data is laid out.


> On a related note, can it harm the hardware if I put a counterfeit game in there by mistake?

For the most part, dumping a counterfeit game shouldn't harm the cart reader. Any cart that could harm the cart reader could also harm the original console.
